packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/userStoryString.st

```diff
@@ -0,0 +1,8 @@
+constants
+userStoryString
+	
+	^ 'AC Calculator User Story
+	   In order to test my sophisticated calculator class
+	   As a developer
+	   I want to do some complicated calculations'
+	"self userStoryName, String cr, self userStoryContent"
```

Comment:
as mentioned in the other pr this is an consistency problem and we should use line 8 and not the hardcoded version